lib/ansible/executor/module_common.py

```diff
@@ -50,6 +55,61 @@
 
 # ******************************************************************************
 
+ZIPLOADER_DATA = u'''%(shebang)s
+# -*- coding: utf-8 -*-'
+import os
+import sys
+import base64
+import tempfile
+
+ZIPDATA = """%(zipdata)s"""
+
+def debug(command, zipped_mod):
+    # Okay to use __file__ here because we're running from a kept file
+    basedir = os.path.dirname(__file__)
+    if command == 'explode':
+        import zipfile
+        z = zipfile.ZipFile(zipped_mod)
+        for filename in z.namelist():
+            if filename.startswith('/'):
+                raise Exception('Something wrong with this module zip file: should not contain absolute paths')
```

Comment:
Should we be raising an Exception here, or returning some JSON/sys.exit(1) as a nice exit response?